app-text/llpp/llpp-30_p20190316.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs xdg-utils
+
+DESCRIPTION="graphical PDF viewer which aims to superficially resemble less(1)"
+HOMEPAGE="https://github.com/moosotc/llpp"
+MY_COMMIT="1f3ae0843d5877a0e599d8411d433bd9b0078157"
+SRC_URI="https://github.com/moosotc/llpp/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
This seems wrong.

Reply:
The license? So it does. 

> This code, sans local copy of LablGL [1], is in the public domain [2]
> 
> [1] Text inside lablGL/COPYRIGHT covers lablGL
> [2] https://github.com/ccxvii/snippets/blob/master/README covers glfont.c
> 

Fixing.